### ISSUE TITLE
Unexpected result

### ISSUE THREAD

**Issue by ScottSmith95** [yunusga/postcss-sort-media-queries#31]
The following input (with version 2.4.17):

```css
@media (min-width: 50em) {
	.colophon-extended li {
		margin-bottom: 2.5rem;
		margin-bottom: var(--space-l3)
	}
}
@media (min-width: 50em) {
	.colophon-extended li img {
		margin-right: 2.5rem;
		margin-right: var(--space-l3)
	}
}
@media (min-width: 50em) {
	.container {
		max-width: 35em;
	}
}
```

Results in the following broken CSS:

```css
@media (min-width: 50em) {
	margin-bottom: 2.5rem;
	margin-bottom: var(--space-l3);
	margin-right: 2.5rem;
	margin-right: var(--space-l3);

	.container {
		max-width: 35em;
	}
}
```

**Comment by yunusga** [comment]
Hi, @ScottSmith95 I was not able to reproduce the problem. I need the source CSS file. It looks like you have a syntax error in CSS somewhere.

**Comment by yunusga** [comment]
@ScottSmith95 i find your problem code. Issue fixed in [v3.0.0](https://github.com/yunusga/postcss-sort-media-queries/tree/v3.0.0)

**Comment by ScottSmith95** [comment]
Ah! OK I was testing both postcss-nested and sort-media-queries and thought the issue might be with this one, not nested. Didn't realize async operation was a thing now. Appreciate your effort!
